code/modules/food/cooking_machines.dm

```diff
@@ -321,7 +321,13 @@ var/global/ingredientLimit = 10
 	cks_max_volume = 400
 	cooks_in_reagents = 1
 
-/obj/machinery/cooking/deepfryer/New()
+/*/obj/machinery/cooking/deepfryer/New()
```

Comment:
Just remove this block. Commented code only needs to exist for review, but this is redundant.